src/JellyfishClientContext.tsx

```diff
@@ -103,21 +103,11 @@ const JellyfishContextProvider = (props: any) => {
     websocket.current = new WebSocket(url);
 
     const processIncomingMessages = new Promise<void>((resolve, reject) => {
-      websocket.current?.addEventListener('open', () => {
-        console.log('WebSocket was opened.');
-      });
-
       websocket.current?.addEventListener('close', (event) => {
         if (event.code !== 1000 || event.isTrusted === false) {
           setError(generateMessage(event));
-          reject(new Error('WebSocket was closed.'));
+          reject(new Error('WebSocket was closed: ' + generateMessage(event)));
```

Comment:
Mby move the 'WebSocked was closed: ' part to the generateMessage function to make it look cleaner here